tcl/util.tcl

```diff
@@ -708,4 +750,133 @@ proc qc::regexp_escape {string} {
     } 
 
     return [string map $list $string]
+ }
+
+proc qc::split2 {args} {
```

Comment:
Does this function essentially convert a string to a list ?

Reply:
Yes

Reply:
Does this do the same thing:-
```Tcl
proc foo {string} {
set result [list]
foreach part $string { lappend result $part }
return $result
}
```

Reply:
Not exactly, this proc takes the delimiter to split on as an argument, the code you've given would use any whitespace as the delimiter.